README.md

```diff
@@ -32,4 +32,36 @@ DB_NAME     = csudh_dev
 ## Documentation
 Access the project documentation through the following links:
 - [Project Overview](https://github.com/sandbox-science/online-learning-platform/wiki/Home)
-- [API Documentation](https://github.com/sandbox-science/online-learning-platform/wiki/API-Doc)
\ No newline at end of file
+- [API Documentation](https://github.com/sandbox-science/online-learning-platform/wiki/API-Doc)
+
+
+## Update Account API
+
+- **Endpoint**: `PUT /update`
```

Comment:
the /update route does not exist. Those are the route your created:

```go
app.Put("/update-username", handlers.UpdateUsername)
app.Put("/update-email", handlers.UpdateEmail)
app.Put("/update-password", handlers.UpdatePassword)
```

So, you want to have documentation for each route.

For instance, for update-username, you will have something like this:

- **Request Body**:
    ```json
    {
        "user_id": "1",
        "username": "myusername"
    }
    ```

 with description, example, and return request like written there https://github.com/sandbox-science/online-learning-platform/wiki/API-Doc